### ISSUE TITLE
Docs for using getStageActor from Java

### ISSUE THREAD

**Issue by johanandren** [akka/akka#24141]
Or possibly a separate Java API, I'm not 100% sure there is a way to call the current `getStageActor((ActorRef, Any) => Unit)` method from Java.

**Comment by ktoso** [comment]
Yeah with Java 8 and Scala 2.12 that'd be a lambda and just work from Java

**Comment by johanandren** [comment]
Except that the higher order function needs to accept `Tuple2<ActorRef, Object>` and return `BoxedUnit`, which isn't very nice from a Java point of view..

**Comment by ktoso** [comment]
Hmm, you're right damn :\

**Comment by johanandren** [comment]
Side note: I wonder why we didn't make it a `(ActorRef, Object) => Unit` instead of a tuple one?

**Comment by chbatey** [comment]
Also add a doc example for Scala
